cli/tools/lint/ast_buffer/ts_estree.rs

```diff
@@ -506,7 +506,7 @@ impl TsEsTreeBuilder {
 
     let kind = match source_kind {
       SourceKind::Module => "module",
-      SourceKind::Script => "script",
```

Comment:
Oops, this seems wrong

```suggestion
SourceKind::Script => "script",
```